engine/pathfinding/private/pathfinding_module.cpp

```diff
@@ -19,10 +19,19 @@ ModuleTickOrder PathfindingModule::Init(MAYBE_UNUSED Engine& engine)
 {
     _renderer = engine.GetModule<RendererModule>().GetRenderer();
 
-    this->SetNavigationMesh("assets/models/NavmeshTest/Navmesh.gltf");
-    ComputedPath path = this->FindPath(
-        { 3.02f, 0.0f, -3.02f },
-        { -3.02f, 0.0f, 3.02f });
+    std::string mesh_path = "assets/models/NavmeshTest/LevelNavmeshTest.glb";
```

Comment:
I know its not the purpose of the PR, but I would try and communicate concrete things like the path from either scripting or the old_engine.hpp file.